<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerMenu</class>
 <widget class="QWidget" name="ComputerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditSearchComputer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetComputer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonAddNewComputer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Computer</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/plus.png</normaloff>:/images/images/plus.png</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonRemoveComputer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Computer</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/minus.png</normaloff>:/images/images/minus.png</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonEditComputer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Computer</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/update.png</normaloff>:/images/images/update.png</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonShowTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Table</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/tafla.png</normaloff>:/images/images/tafla.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelType">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelWasBuilt">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Was Built:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYearBuilt">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Year Built:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComputerName">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Computer Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComputerType">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Computer Type</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxWasBuilt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Was it Built?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComputerYearBuilt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Year Built</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCllose">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>83</red>
         <green>83</green>
         <blue>83</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>83</red>
         <green>83</green>
         <blue>83</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
